master/internal/experiment.go

```diff
@@ -1135,3 +1137,12 @@ func (e *internalExperiment) setRP(resourcePool string) error {
 
 	return nil
 }
+
+func notASearcherShutdown(ops []searcher.Operation) bool {
```

Comment:
I think this function name made it harder to understand the intent of the function and how it was used. 

What if the function was named something like `allSearcherShutdown(...)` and it returned `false` as soon as it found an operation that wasn't a shutdown error (and otherwise `true`)?

Then the if statement above would look like 
```
if _, ok := model.StoppingStates[e.State]; ok && !allSearcherShutdown(ops) {
```

Just my 2cents. Feel free to ignore if this sounds more confusing to you.

Reply:
yeah this is what I was afraid of, thanks for the feedback